docs/getting-started/README.md

```diff
@@ -1,31 +1,31 @@
 # 🚀 Getting started
 
-Welcome! 👋
+👋 Welcome! You've just taken your first step on the path to mastering Frontity.
 
-Frontity is a **React framework** to create **WordPress themes**. It allows you to easily build a React frontend for a headless WordPress site, which serves its data via the WP REST API. Frontity framework is open source and free to use.
+Frontity is a **React-based framework** that enables you to easily build a frontend for a headless (or decoupled) WordPress site. Your WordPress site serves its data via the REST-API. Frontity framework is open source and free to use.
 
-## Get started
+## Installation requirements
 
-You can get started right now, it will be really easy! You may check:
+To get started with Frontity you will need:
 
-* 📘 [**Quick start guide**](quick-start-guide.md)**:** here you will find how to create a website with Frontity, step by step.
-* 👨‍👩‍👧‍👦 [**Community forum**](https://community.frontity.org/)**:** the perfect place to ask any questions and share your feedback.
+* **A WordPress installation** - This can be hosted locally, on a web-server, or you can also use a site hosted on wordpress.com. However, when you spin up your first Frontity project you will initially be connected to our test site so you won't be without content when you first try Frontity.
```

Comment:
I would add here the pattern URL where the REST API is available:
- For a custom "wordpress.org" installation → http://your-wp-domain.com/wp-json
- For a "wordpress.com"  → https://public-api.wordpress.com/wp/v2/sites/your-wp-site.wordpress.com

Reply:
I'm not sure if these details should be under 'Installation Requirements'. Here we're just telling them what they need prior to running npx frontity create. I can add this information on the new page 'Connecting Frontity to Your WordPress Site' where I think it will be more appropriate. What do you think?

Reply:
Well, you actually need a working REST API URL to "get started" with Frontity. You cannot start working with Frontity:
- If your Wordpress installation have an old version and doesn't provide REST API
- If your Wordpress installation is failing and you're getting an error

So I would provide these "pattern links" as a way to check that your WP installation is ready to be connected with your Frontity app

Reply:
Added.  👍